services/userService.js

```diff
@@ -0,0 +1,24 @@
+const bcrypt = require("bcrypt");
+const response = require("express");
+const jwt = require("jsonwebtoken");
+
+const userDao = require("../models/userDao");
+
+const signIn = async (email, password) => {
+  const hashedPassword = await userDao.getHashedPassword(email);
+  const compare = await bcrypt.compare(password, hashedPassword);
+
```

Comment:
유저가 입력한 이메일이 유효하지 않은 경우도 검증해야 합니다. 
이메일이 잘못된 건지, 패스워드가 잘못된건지 에러메세지를 따로 작성할 수도 있고, 동일하게 `INVALID_USER_DATA`와 같은 식으로 작성될 수 있지만 이는 기획적으로 선택해주시면 될 것 같습니다.
(친절하게 뭐가 잘못된 건지 따로 알려주는 경우는 보안적으로 취약하겠죠?)

```suggestion
  const hashedPassword = await userDao.getHashedPassword(email);
  
  if(!hashedPassword) {} 
  
  const compare = await bcrypt.compare(password, hashedPassword);

```